### DISCUSSION TITLE
Server Error when Removing Remote Auth Configuration and Trying to Log In with a Local User

### DISCUSSION THREAD

**Discussion by r0bc94** [netbox-community/netbox#14630] [Help Wanted!] [answered]
In Netbox 3.6.5, I wanted to remove a former configured remote authentication. To remove such config, I simply commented out all the lines stating something with `REMOTE_AUTH`. Howerver, now I'm not able to login anymore with my local user. 

When trying to log in with the local user, a 500 error is returned by netbox stating the following error message: 

```
<class 'netaddr.core.AddrFormatError'>

failed to detect a valid IP address from ''

Python version: 3.8.10
NetBox version: 3.6.5
Plugins: 
  netbox_dns: 0.21.3
  netbox_interface_sync: 0.2.0
  netbox_qrcode: 0.0.10
  netbox_topology_views: 3.8.1
```
It would be great if you could help me finding the issue here. 

Best regards

**Comment by candlerb** [comment, marked as answer]
Updating to 3.6.7 or later is likely to fix your problem.

* [#14249](https://github.com/netbox-community/netbox/issues/14249) - Fix server error when authenticating via IP-restricted API tokens using IPv6
